src/render/src/lib.rs

```diff
@@ -64,7 +64,7 @@ mod encoder;
 mod factory;
 /// Meshes
 mod mesh;
-/// Pipeline states
+// Pipeline states
```

Comment:
This was intentional. Adding the extra / there causes it to merge with the `//!` style comments in the module itself.